src/apps/material/order-digital-copy.test.ts

```diff
@@ -134,6 +144,55 @@ describe("Material - Order digital copy", () => {
       .and("contain", "Close")
       .click();
   });
+
+  beforeEach(() => {
+    cy.interceptGraphql({
+      operationName: "getMaterial",
+      fixtureFilePath: "material/order-digital-copy/order-digital-fbi-api"
+    });
```

Comment:
Please reconsider this `beforeEach` implementation.

To me it looks like what is already defined on L4 in this file. If they are different then one has to go.

Perhaps this is a merge conflict gone wrong?

Reply:
Nice spotted!
I have corrected it.